agrona/src/main/java/org/agrona/concurrent/status/AtomicCounter.java

```diff
@@ -240,6 +261,25 @@ public long decrement()
      * @return the previous value of the counter
      */
     public long decrementOrdered()
+    {
+        return decrementRelease();
+    }
+
+    /**
+     * Decrements the counter non-atomically.
+     * <p>
+     * It can result into lost updates to race condition when called concurrently.
+     * <p>
+     * The load has plain memory semantics and the store has release memory semantics.
+     * <p>
+     * The typical use-case is when there is one mutator thread, that calls this method, and one or more reader threads.
+     * <p>
+     * This method is likely to outperform the {@link #increment()} and probably will be a better alternative.
+     *
+     * @return the previous value of the counter
+     * @since 2.1.0
+     */
+    public long decrementRelease()
```

Comment:
Call `org.agrona.UnsafeApi#getAndAddLongRelease` with `-1` to ensure that decrement is atomic.